core/templates/pages/collection-player-page/collection-local-nav/collection-local-nav.component.html

```diff
@@ -0,0 +1,15 @@
+<div>
+  <ul class="nav navbar-nav oppia-navbar-nav float-right oppia-navbar-pencil-icon">
+    <li *ngIf="canEdit" class="nav-item">
+      <a [href]="'/collection_editor/create/' + collectionId" uib-tooltip="Edit" tooltip-placement="bottom" target="_blank" rel="noopener" class="nav-link">
```

Comment:
Extending on this [comment](url). This is the change I am expecting.
```suggestion
      <a [href]="'/collection_editor/create/' + collectionId" ngbTooltip="Edit" placement="bottom"  target="_blank" rel="noopener" class="nav-link">
```